app/components/course-page/course-stage-step/tests-passed-modal/index.ts

```
@@ -0,0 +1,44 @@
+  Args: {
+    onClose: () => void;
+    currentStep: CourseStageStep;
+  };
+}
+
+export default class TestsPassedModalComponent extends Component<Signature> {
+  @service declare courseStageCompletion: CourseStageCompletionService;
+
+  @tracked action: 'choose_action' | 'refactor_code' | 'mark_stage_as_complete' | 'submit_code' = 'choose_action';
+
+  @action
```

Biome (1.9.4)

[error] This parameter is used before its declaration.

The parameter is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)